### ISSUE TITLE
Update All Available Versions in HestiaUI Documentations

### ISSUE THREAD

**Issue by CoryGalyna** [ZORALab/Hestia#73]
# Description
```
Please provide a short description of what you have encountered below.
```
All the HestiaUI Documentations are recording `v0.0.1` as its base release version while the actual one is `v1.0.0`. 

# Link
```
Please provide the documentation link(s) below.
```
All pages inside `/en/docs/hestiaui/` path.

# Expected Presentation
```
Please specify the expected presentation of the documentation.
```
base version should be `v1.0.0`.

# Current Presentation
```
Please specify the current presentation of the documentation.
```
base version is `v0.0.1` which does not exist.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Ouch.

**Comment by CoryGalyna** [comment]
Enjoy.

**Comment by hollowaykeanho** [comment]
Implemented in fac80033936009ea13b8dbc37749a1ea38f9e386

> Enjoy.

Not that hard with vim. =)

**Comment by hollowaykeanho** [comment]
published.
